lib/urlwatch/jobs.py

```diff
@@ -230,3 +231,29 @@ def retrieve(self, job_state):
                 return response.content.decode('ascii', 'ignore')
 
         return response.text
+
+class FileJob(Job):
+    """Read the content of a local file"""
+
+    __kind__ = 'file'
+    __required__ = ('path',)
+
+    def get_location(self):
+        return str(self.path)
+
+    def retrieve(self, job_state):
+
+        path_string = str(self.path)
+
+        if not path_string:
+            raise ValueError('Path cannot be null or empty')
+
+        if path_string.startswith('file://'):
```

Comment:
This pull request is only for local files, adding special-casing for `file://` URIs should be enough. No need to deal with alternative protocols such as `ftp://`. URLs starting with `file://` get special treatment (we /do/ know that there's no point in giving `file://` to `requests.request()`), and everything else will be passed to requests, whether it works or not.